scripts/write_elmo_representations_to_file.py

```diff
@@ -0,0 +1,137 @@
+"""
+Given a pre-processed input text file, this script dumps all of the internal
+layers used to compute ELMo representations to a single (potentially large) file.
+
+The input file is pre-tokenized whitespace separated text, one sentence per line.
+The output is a hdf5 file where each sentence is a size (3, num_tokens, 1024)
```

Comment:
I googled for what hdf5 was.  The first results were not very helpful.  Eventually I found `http://docs.h5py.org/en/latest/` which satisfied me for now.  I'd consider adding a little more detail about the output format, or linking to `http://docs.h5py.org/en/latest/`.